.travis.yml

```diff
@@ -8,6 +8,8 @@ cache:
     - ${HOME}/cached-deps
     - ${HOME}/gopath/pkg/mod
     - "$(python3 -c 'import site; print(site.USER_BASE)')"
+    - ${HOME}/docker_images
```

Comment:
Docker image caching was scrapped [in this PR](https://github.com/pachyderm/pachyderm/pull/5007) for two reasons:
1) The cache would slowly leak and increase in size, eventually creating cache assets that are hundreds of gigabytes in size and triggering timeouts.
2) Caches aren't shared between the build step and test steps.

Reply:
OK, useful context thanks! I've moved to trying to only cache specific named docker images to improve build times, but that may be futile, and I'm close to ripping it back out :) The main thrust of this PR will be retries (reliability) rather than speed, but it's always tempting to work on improving the speed of CI while waiting for it to run!